packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -126,6 +135,35 @@ export const workflowsRouter = createProtectedRouter()
       }
     },
   })
+  .mutation("createV2", {
```

Comment:
New mutation to create workflow as we decided to remove the modal in V2 when creating a workflow